tools/python/boutiques/localExec.py

```diff
@@ -96,18 +95,18 @@ def execute(self):
       # Prepare extra environment variables
       envString = " "
       if envVars:
-        for (key,val) in envVars.items(): envString += "-e " + str(key) + "=\'" + str(val) + '\' '
+        for (key,val) in list(envVars.items()): envString += "-e " + str(key) + "=\'" + str(val) + '\' '
       # Change launch (working) directory if desired
       launchDir = '${PWD}' if (self.launchDir is None) else self.launchDir
       # Run it in docker
       dcmd = 'docker run --rm' + envString + ' -v ${PWD}:' + launchDir + ' -w ' + launchDir + ' ' + str(dockerImage) + ' ./' + dsname
-      print('Executing in Docker via: ' + dcmd)
+      print(('Executing in Docker via: ' + dcmd))
       exit_code = self._localExecute( dcmd )
     # Otherwise, just run command locally
     else:
       exit_code = self._localExecute( command )
     # Report exit status
-    print('---/* End program output */---\nCompleted execution (exit code: ' + str(exit_code) + ')')
+    print(('---/* End program output */---\nCompleted execution (exit code: ' + str(exit_code) + ')'))
```

Comment:
That was a mistake in the 2to3 conversion that I failed to catch - I'll remove the second set :) Thanks!